### DISCUSSION TITLE
Updated DATETIME field in hooks returns error.

### DISCUSSION THREAD

**Discussion by AlexVanK** [directus/directus#10140] [Archive]
This is my code:

```JS
   const tzoffset = new Date().getTimezoneOffset() * 60000; //offset in milliseconds
   const localISOTime = new Date(Date.now() - tzoffset).toISOString().slice(0, -1);
   const fullDate = localISOTime.slice(0, 19).replace("T", " ");
```
which generates a time string in the MySQL format: `2021-11-29 12:07:59`

I am trying to update this via a custom HOOK:

```JS
   input["date_pending"] = fullDate;
   return input;
```

But the console is throwing an error:
![image](https://user-images.githubusercontent.com/56647386/143917975-40f6322c-6781-4eb2-be27-53347b186b5b.png)

Notice that the datetime string is generated correctly. What format do I have to pass the date time string as, for it to make it through
the `processDates(payloads, action)` service?

**Comment by joselcvarela** [comment]
Hello @AlexVanK 
As you can see here https://github.com/directus/directus/blob/76a306e7f434638d14c5bdf7cc946b4b657a614b/api/src/services/payload.ts#L314-L322

We already handle dates, so you should pass a timestamp like `Date.now()` our a standard date string like `new Date().toISOString()`. In your case you are passing a custom date, which can't be handled.

**Reply by AlexVanK** [reply]
I ended up passing an instance of 
`new Date()`
and it works!

Thank you :)
